tests/impl.cpp

```diff
@@ -3117,7 +3117,25 @@ result_t test_mm_blendv_epi8(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_slli_epi32(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    const int32_t *_a = (const int32_t *) impl.mTestIntPointer1;
+    const int32_t *_b = (const int32_t *) impl.mTestIntPointer2;
+#if defined(__clang__)
+    int count = (uint32_t) _b[0] % 32;
```

Comment:
Could you add the comment describing the reason for using `32` with the `Clang` compiler?
I remember that it is the compiler error which indicates that the value larger than `32` would out of the shift range.